app/views/welcome/index.html.erb

```diff
@@ -32,8 +38,59 @@
           </select>
         </div>
 
+        <div class="form-group">
+          <label class="form-label radio-label" for="info_basic">
+            <%= radio_button_tag("info", "basic", true, class: "info-basic") %>
+            <span class="radio-tag">Basic Information</span>
+            <div class="control-indicator" ></div>
+          </label>
+        </div>
+
+        <div class="form-group">
+          <label class="form-label radio-label" for="info_enhanced">
+            <%= radio_button_tag("info", "enhanced", false, class: "info-enhanced") %>
+            <span class="radio-tag">Enhanced Information</span>
+            <div class="control-indicator" ></div>
+          </label>
+        </div>
+
+        <div class="enhanced-checkbox">
+          <% %w(
+            content_id
+            document_collections
+            is_historic
+            mainstream_browse_pages
+            organisations
+            people
+            policies
+            popularity
+            taxons
+            specialist_sectors
+          ).each do |option| %>
```

Comment:
Don't we have this list as a constant somewhere?

Reply:
we do now